cert.go

```diff
@@ -144,7 +144,25 @@ func loadCAFromConfig(l *logrus.Logger, c *config.C) (*cert.NebulaCAPool, error)
 		}
 	}
 
+	var expired int
 	CAs, err := cert.NewCAPoolFromBytes(rawCA)
+	if errors.Is(err, cert.ErrExpired) {
+		for _, cert := range CAs.CAs {
+			if cert.Expired(time.Now()) {
+				expired++
+				fingerprint, _ := cert.Sha256Sum()
+				l.WithField("name", cert.Details.Name).
+					WithField("not_before", cert.Details.NotBefore).
+					WithField("not_after", cert.Details.NotAfter).
+					WithField("fingerprint", fingerprint).
+					WithField("signature", fmt.Sprintf("%x", cert.Signature)).
+					Warn("expired certificate present in CA pool")
+			}
+		}
+		if expired < len(CAs.CAs) {
+			err = nil
+		}
+	}
 	if err != nil {
```

Comment:
An example output running with all expired CAs:

```
WARN[0000] expired certificate present in CA pool        fingerprint=152070be6bb19bc9e3bde4c2f0e7d8f4ff5448b4c9856b8eccb314fade0229b0 name=expired not_after="2021-11-29 23:55:05 -0600 CST" not_before="2021-11-29 23:55:04 -0600 CST" signature=6e2a967662f039dcbc3fcb0d99cb8091ce697d435821c7121e7d7a956c0be04d81389e587d4cf67e0eedc16d1a533ab0924e4beac2555109dc0052efe30f750b
ERRO[0000] Failed to load ca from config                 error="error while adding CA certificate to CA trust store: certificate is expired"
```

We should return a better error in this case, something like "no valid CA certificates present" while leaving this one for the non expired certs case

If there aren't any valid certificates and it's  `cert.ErrExpired` and if it is, shift to logging something like "no valid CA certificates present